api/src/main/java/io/strimzi/api/kafka/model/template/JmxTransTargetQueryTemplate.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.template;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.api.kafka.model.UnknownPropertyPreserving;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * The {@code spec} of a {@link io.strimzi.api.kafka.model.Kafka}.
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "kafka", "zookeeper", "topicOperator",
+        "entityOperator", "clusterCa", "clientsCa",
+        "maintenance"})
+@EqualsAndHashCode
+public class JmxTransTargetQueryTemplate implements Serializable, UnknownPropertyPreserving {
```

Comment:
This class does not seem to be used anywhere and seems to have no actual fields. What is the purpose of it?